api/api.pb.go

```diff
@@ -16,7 +16,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.20.1
```

Comment:
Ooo - do we need to update any docs/deps for this protoc version change?

Reply:
Do you mean from a transparency perspective? I think we didn't put this version anywhere as a variable because the git commit captures the version of protoc / the generated files at any point in time.

I also searched in our transparency-dev org; the other thing using protoc is the [verifier](https://github.com/search?q=org%3Atransparency-dev+protoc&type=code).

Reply:
It was more for churn - e.g. will these generated files "flap" between slightly different output from different versions of the tools as we all continue working in here. In Trillian there's a pinned version of `protoc` (+ support) and in the CI builds it explicitly runs `go generate` and looks for differences to enforce that everyone is using the same versions of the tooling.

Reply:
Ah, got it. Filed an issue here: https://github.com/transparency-dev/armored-witness-os/issues/157